tests/test_fair_estimations.py

```diff
@@ -122,62 +122,51 @@ def linear_extrapolation(x, x_1, x_2, y_1, y_2):
     )
 
 
-def test_c_equivalence():
+@pytest.mark.parametrize(
+    "eta, expected",
+    [
+        pytest.param(0, xr.DataArray(np.array(5.0)), id="eta=0"),
+        pytest.param(1, xr.DataArray(np.array(4.16179145)), id="eta=1"),
+        pytest.param(10, xr.DataArray(np.array(2.32634448)), id="Normal eta"),
```

Comment:
what does "normal eta" mean?

Reply:
eta of 0 and 1 might be cases where the original test was looking for specific behavior. eta=10 didn't really seem special so it's a basic, "normal" case... Happy to call it something else if you have ideas, @kemccusker

Reply:
cool, that makes sense. Can we just call it eta=10?